src/test/java/com/facebook/openwifirrm/modules/aggregators/MeanAggregatorTest.java

```diff
@@ -0,0 +1,45 @@
+package com.facebook.openwifirrm.modules.aggregators;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+public class MeanAggregatorTest {
+
+	private static double eps;
```

Comment:
do you envision many more tests in this class, that would need a global `eps`?

Reply:
No, actually I can move it into the method. It can be moved back out later (perhaps to even a parent class) if it turns out there are other aggregators and they all want to share the same `eps`